### PR TITLE
Remove RPC

### PR DESCRIPTION
**Pull request by chhsiao1981**
1. After discussion, we think that it's good to separate the api-call and frontend-page-call.
2. The ethereum-rpc provides good json-rpc api-call.
3. The ethereum-rpc does not provide httpServer and cannot restart appropriately.
4. We impleemnt api.endpoints to be able to do restart.
5. For multi-part http-post, currently we workaround by integrating with frontend-page-call.
   We may integrate into api.endpoints in the future.

### CODE REVIEW

File: api/endpoints.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 The go-pttai Authors
```
Comment (poga) [inline api/endpoints.go]: so this is basically copied from the previous version of go-ethereum?

File: api/endpoints.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+import (
+    "net"
+    "net/http"
+
+    "github.com/ethereum/go-ethereum/rpc"
+)
+
+// StartHTTPEndpoint starts the HTTP RPC endpoint, configured with cors/vhosts/modules
+func StartHTTPEndpoint(endpoint string, apis []rpc.API, modules []string, cors []string, vhosts []string, timeouts rpc.HTTPTimeouts) (net.Listener, *rpc.Server, *http.Server, error) {
```
Comment (poga) [inline api/endpoints.go]: If this is only for JSON RPC, maybe rename the file to `rpc/endpoints.go`?

Comment (chhsiao1981) [reply api/endpoints.go]: renamed api as rpc.

File: api/endpoints.go (hunk 1 above)
Comment (chhsiao1981) [reply api/endpoints.go]: yes～

**Review by poga — APPROVED** [review]

**Pull request merged**